monasca-statsd/README.md

```diff
@@ -0,0 +1,68 @@
+monasca-statsd Dockerfile
+========================
+
+This image contains a containerized version of the Monasca statsd. For
+more information on the Monasca project, see [the wiki][1]. It is based on the
+agent-base image also built in monasca-docker.
+
+Sources: [monasca-statsd][2] &middot; [monasca-docker][3] &middot; [Dockerfile][4]
+
+Tags
+----
+
+Images in this repository are tagged as follows:
+
+ * `latest`: refers to the latest stable point release, e.g. `1.6.0`
+ * `1.6.0`, `1.6`, `1`: standard semver tags, based on git tags in the
+   [official repository][2].
+ * `mitaka`, `newton`, etc: named versions following OpenStack release names
+   built from the tip of `stable/RELEASENAME` branches in the repository
+ * `master`, `master-DATESTAMP`: unstable testing builds from the master branch,
+   these may have features or enhancements not available in stable releases, but
+   are not production-ready.
+
+Usage
+-----
+
+In environments resembling the official [docker-compose][3] or [Kubernetes][4]
+environments, the image does not require additional configuration parameters and
+can be minimally run like so:
+
+    docker run -it monasca/statsd:latest
+
+Configuration
+-------------
+
+| Variable                    | Default          | Description                         |
```

Comment:
Could you format this table to be easier to read in code?
Like using https://www.tablesgenerator.com/markdown_table (Goto `File` => `Paste table data...`)